pkg/generators/models/templates.go

```diff
@@ -76,7 +76,9 @@ func (m {{$modelName}}) Validate() error {
 					{{- if .IsBase64 }}is.Base64,{{ end }}
 					{{- if .IsEmail }}is.EmailFormat,{{ end }}
 					{{- if .IsUUID }}is.UUID,{{ end }}
-					{{- if .IsURL }}is.RequestURL,{{ end }}
+					{{- if .IsURL }}is.URL,{{ end }}
+					{{- if .IsURI }}is.RequestURI,{{ end }}
+					{{- if .IsRequestURI }}is.RequestURL,{{ end }}
```

Comment:
I think these error messages improve the final errors and align a little better with our existing validation in hub
```suggestion
					{{- if .IsURL }}is.URL.Error("must be a valid URL with HTTP or HTTPS scheme"),{{ end }}
					{{- if .IsURI }}is.RequestURI,{{ end }}
					{{- if .IsRequestURI }}is.RequestURL.Error("must be valid URI with scheme"),{{ end }}
```